lib/paypal_express/api.rb

```diff
@@ -113,10 +113,13 @@ def add_payment_method(kb_account_id, kb_payment_method_id, payment_method_props
           }
         else
           # Go to Paypal to get the Payer id (GetExpressCheckoutDetails call)
-          payer_id = find_payer_id(token, kb_account_id, context.tenant_id, properties_to_hash(properties))
+          payment_processor_account_id = find_value_from_properties(properties, :payment_processor_account_id)
+          payment_processor_account_id ||= find_payment_processor_id_from_initial_call(kb_account_id, context.tenant_id, token)
```

Comment:
I see line 270 below that we set `:default` as a default. Shouldn't we do the same here?